workloads/ric/pkg-example-ric/Kptfile

```diff
@@ -0,0 +1,18 @@
+apiVersion: kpt.dev/v1
+kind: Kptfile
+metadata:
+  name: pkg-ric
+  annotations:
+    config.kubernetes.io/local-config: "true"
+info:
+  description: example ric blueprint package
+
+pipeline:
+  mutators:
+  - image: gcr.io/kpt-fn/apply-replacements:v0.1.1
+    configPath: apply-replacements-owner.yaml
+  - image: gcr.io/kpt-fn/apply-replacements:v0.1.1
+    configPath: apply-replacements-namespace.yaml
+  - image: gcr.io/kpt-fn/set-namespace:v0.4.1
+    configPath: package-context.yaml
+  - image: nephio/nfdeploy-fn:v2.0.0
```

Comment:
@naman-gupta-641 Do you need this pipeline? Are you creating NAD?

Reply:
yeah as per discussion , I removed it since i am not changing NFdeployment as of now, will be required when i will be adding E2 interface support.